install_config/persistent_storage/dynamically_provisioning_pvs.adoc

```diff
@@ -126,11 +126,17 @@ from plug-in to plug-in.
 
 To set a _StorageClass_ as the cluster-wide default:
 ----
-   storageclass.beta.kubernetes.io/is-default-class: "true"
+   storageclass.kubernetes.io/is-default-class: "true"
 ----
 This enables any Persistent Volume Claim (PVC) that does not specify a specific
 volume to automatically be provisioned through the _default_ StorageClass
 
+[NOTE]
```

Comment:
Looks like we need another = per the [guidelines](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#admonitions). Thanks!